src/lib.rs

```diff
@@ -283,6 +283,7 @@ pub fn from_primitive(input: TokenStream) -> TokenStream {
                     <#inner_ty as #import::FromPrimitive>::from_i32(n).map(#name)
                 }
                 #[inline]
+                #[cfg(has_i128)]
```

Comment:
This `cfg` is being emitted into the user's code, so I think it will effectively block all `i128` unless *they* happen to have set `has_i128` in the target crate.